invenio/base/bundles.py

```diff
@@ -133,11 +134,23 @@
 
 admin = Bundle(
     "js/admin.js",
+    "vendors/admin-lte/dist/js/app.min.js",
     output="admin.js",
     filters=RequireJSFilter(exclude=[jquery]),
     weight=50
 )
 
+adminnextgen = Bundle(
+    "vendors/admin-lte/dist/css/AdminLTE.min.css",
+    "vendors/admin-lte/dist/css/skins/{0}.min.css".format(current_app.config.get("ADMIN_UI_SKIN")),
+    output="admin-lte.css",
```

Comment:
No, but wanted to separate the two styles in their names, otherwise it would be `/css/admin.css` and `/gen/admin.css` (for new styles). But can change it to `admin.css`.